binds/python/bind_immutable.cpp

```diff
@@ -149,14 +149,130 @@ void bind_immutable_module(py::module& m) {
             "- morphio.IterType.breadth_first (default)\n"
             "iter_type"_a = IterType::DEPTH_FIRST);
 
-    py::class_<morphio::GlialCell, morphio::Morphology>(m, "GlialCell")
-        .def(py::init<const std::string&>())
-        .def(py::init([](py::object arg) {
-                 return std::unique_ptr<morphio::GlialCell>(new morphio::GlialCell(py::str(arg)));
+    py::class_<morphio::GlialCell>(m, "GlialCell")
```

Comment:
After having discussed this point with Benoit, it appears that it is not doable because different Type are also used morphio::Morphology vs morphio::GlialCell for example.